test/testlib/iplookup.go

```diff
@@ -13,5 +10,17 @@ import (
 
 // LookupIP looks up the IP address of the provided hostname, preferring IPv4.
 func LookupIP(ctx context.Context, hostname string) ([]net.IP, error) {
-	return net.DefaultResolver.LookupIP(ctx, "ip4", hostname)
+	ips, err := net.DefaultResolver.LookupIPAddr(ctx, hostname)
+	if err != nil {
+		return nil, err
+	}
+
+	// Filter out to only IPv4 addresses
```

Comment:
Fine because we only care about IPv4 in testing scenerios?

Reply:
The code is brought in from `test/testlib/iplookup_go1.14.go` which is the implementation for any version of go newer than 1.13.